spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/headers.adoc

```diff
@@ -179,3 +179,81 @@ MessagingMessageConverter converter() {
 
 If using Spring Boot, it will auto configure this converter bean into the auto-configured `KafkaTemplate`; otherwise you should add this converter to the template.
 
+[[multi-value-header]]
+== Support multi-value header
+
+Spring for Apache Kafka 4.0 supports multi-value header where same logical header key appears more than once in a Kafka record.
```

Comment:
Usually we say `Starting with 4.0,`. Otherwise it sounds like in `5.0` there won’t be such a support 😜.
Plus no need to mention our project name in every sentence since it is kinda obvious that this doc is about Spring for Apache Kafka.